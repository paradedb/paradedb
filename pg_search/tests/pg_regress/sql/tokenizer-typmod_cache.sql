DELETE FROM paradedb._typmod_cache WHERE typmod = ARRAY['66', '77'];
BEGIN;
SELECT 'hello, world'::pdb.ngram(66, 77)::text[];
ABORT;
SELECT 'hello, world'::pdb.ngram(66, 77)::text[];
