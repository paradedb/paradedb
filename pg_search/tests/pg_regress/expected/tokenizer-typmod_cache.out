DELETE FROM paradedb._typmod_cache WHERE typmod = ARRAY['66', '77'];
BEGIN;
SELECT 'hello, world'::pdb.ngram(66, 77)::text[];
 text 
------
 {}
(1 row)

ABORT;
SELECT 'hello, world'::pdb.ngram(66, 77)::text[];
 text 
------
 {}
(1 row)

